_topic_map.yml

```diff
@@ -50,6 +50,10 @@ Topics:
 - Name: Product architecture
   File: architecture
   Distros: openshift-enterprise,openshift-origin,openshift-dedicated
+- Name: Architecture walkthrough
+  File: introduction-openshift-architecture
+- Name: Understanding OpenShift development
+  File: understanding-openshift-development
 - Name: Abstraction layers and topology
   File: architecture-topology
 - Name: Installation and update
```

Comment:
Check for duplication and remove.
Same for `architecture-operators`

customizations needs to stay